frontend/src/app/components/tasks-filters/tasks-filters.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, inject, OnInit ,EventEmitter, Output} from "@angular/core";
+import { TaskStatus } from "../../models/task-status";
+import { AppStore } from "../../store/app.store";
+import { AppState } from "../../models/app-state";
+import { FormControl } from "@angular/forms";
+
+@Component({
+  selector: 'app-tasks-filters',
+  templateUrl: 'tasks-filters.component.html',
+  styleUrls: ['tasks-filters.component.scss']
+})
+export class TasksFiltersComponent implements OnInit {
+  protected TaskStatus = TaskStatus;
+  protected appStore: AppStore = inject(AppStore);
+  protected currentStatus: TaskStatus;
```

Comment:
I suggest renaming `currentStatus` to `currentTaskStatus` for better clarity. Use `Shift + F6` for that.

Reply:
Done